src/modules/villages/village_service.ts

```diff
@@ -50,4 +52,32 @@ export namespace Village {
 
     return data
   }
+
+  export const findById = async ({ id }: { id: string }): Promise<Entity.ResponseFindById> => {
+    const item = await Repository.findById(id)
+
+    if (!item) {
+      throw new HttpError(httpStatus.NOT_FOUND, `village with id ${id} does not exits`)
+    }
+
+    const data = {
```

Comment:
please add interface ResponseFindById 
`
{
data: {},
meta: {}
}
`

then make variable result implement interface ResponseFindById

Reply:
https://github.com/jabardigitalservice/dedi-core-service/blob/4cc9b6270711c3aee52cc8e1147e07a972e4964f/src/modules/villages/village_entity.ts#L60-L63

this one?

Reply:
yups :+1:

Reply:
it's already implemented then